src/bot/common/court_resolver.py

```diff
@@ -55,8 +56,15 @@ async def resolve_court_address(city: str, court_subj: str, street: str) -> List
                       "Chrome/94.0.4606.81 Safari/537.36"
     }
 
-    async with ClientSession() as session:
-        async with session.get(url, headers=headers, ssl=False,) as resp:
-            body: str = await resp.text()
-            result = parse_court_data(body, city)
-            return result
+    try:
+        async with ClientSession() as session:
+            # For better user experience setup request timeout to 15 seconds
+            timeout = ClientTimeout(total=15)
+            async with session.get(url, headers=headers, ssl=False, timeout=timeout) as resp:
+                body: str = await resp.text()
+                result = parse_court_data(body, city)
+                return result
+    except Exception:
+        logger = logging.getLogger()
+        logger.error("Error occurred during court address resolving")
```

Comment:
Since we catch any exception here, let's log the exception details as well

Reply:
@nchursin So, the `logger.exception` [includes ](https://docs.python.org/3/library/logging.html#logging.Logger.exception)info about exception into the logging message. But I used `logger.error` - this should be fixed :)